themes/ddbasic/gulpfile.js

```diff
@@ -82,6 +82,28 @@ gulp.task('sass', 'Process SCSS using libsass',
 );
 
 /**
+ * Usage:
+ * gulp sassprod
+ *
+ * Precompile all sass files into one single bundle.css file in the sass_css folder
+ *
+ */
+gulp.task('sassprod', 'Process SCSS into one css file using libsass',
```

Comment:
The naming of this task `sassprod` is odd here.

1. As far as I can tell the resulting `bundle.css` is not used for production purposes in the rest of the codebase.
2. If it is to be used for production purposes I would have expected `outputStyle: 'compact'`
3. If it is only used by KSS I would expected it to be called by the `kss` task. On the other hand it actually seems to assemble `bundle.css` on its own.

Perhaps we can delete the task entirely?

Reply:
To keep styling and documentation apart the tasks are seperate. I have clarified in the comments and changed the gulp task name. Let me know if you prefer to unify the tasks.

Reply:
I would prefer that the tasks are unified. The result of `ksssass` is not used anywhere else.